### PR TITLE
log a warning when sending requests with a large body of bytes

### PR DESCRIPTION
**Pull request by arthurdarcet**
As discussed in #2126 , log a warning when sending a request with a large body payload passed as bytes directly: aiohttp does not do any chunking for this use case, and it can freeze the event loop.

### CODE REVIEW

File: aiohttp/payload.py [hunk 1]
```diff
@@ -150,6 +151,11 @@ def __init__(self, value, *args, **kwargs):
 
         self._size = len(value)
 
+        if self._size > DEFAULT_LIMIT:
```
Comment (fafhrd91) [inline aiohttp/payload.py]: I think `DEFAULT_LIMIT` is too small. lets define new const, something close to 1mb

Comment (arthurdarcet) [reply aiohttp/payload.py]: ok, do you want it in `payload.py` or in `stream.py` next to the other constants?

Comment (fafhrd91) [reply aiohttp/payload.py]: it should be payload specific

**Comment by fafhrd91** [comment]
thanks

**Pull request merged**